### PR TITLE
Make it possible to choose socket file

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: scripts/plutus/simple-minting-policy.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ set -o pipefail
 
 # Step 1 - Send ADA to token script owner
 
-export CARDANO_NODE_SOCKET_PATH=example/node-bft1/node.sock
+export CARDANO_NODE_SOCKET_PATH="${CARDANO_NODE_SOCKET_PATH:-example/node-bft1/node.sock}"
```
Comment (newhoggy) [inline scripts/plutus/simple-minting-policy.sh:16]: Provide a default value, but allow the caller of the script to define another value.

**Comment by newhoggy** [comment]
Example run:

```
$ CARDANO_NODE_SOCKET_PATH=test.socket WORK="/private/var/folders/zh/ln41q4zs52x2fd61rxccmq640000gn/T/chairman/test-96968fe06fa63acf" UTXO_VKEY=/private/var/folders/zh/ln41q4zs52x2fd61rxccmq640000gn/T/chairman/test-96968fe06fa63acf/shelley/utxo-keys/utxo1.vkey UTXO_SKEY=/private/var/folders/zh/ln41q4zs52x2fd61rxccmq640000gn/T/chairman/test-96968fe06fa63acf/shelley/utxo-keys/utxo1.skey ./scripts/plutus/example-txin-locking-plutus-script.sh
Always succeeds Plutus script in use. Any datum and redeemer combination will succeed.
Script at: scripts/plutus/scripts/untyped-always-succeeds-txin.plutus
Transaction successfully submitted.
Pausing for 5 seconds...
Submit the tx with plutus script and wait 5 seconds...
Transaction successfully submitted.

Querying UTxO at addr_test1vpqgspvmh6m2m5pwangvdg499srfzre2dd96qq57nlnw6yctpasy4. If there is ADA at the address the Plutus script successfully executed!

                           TxHash                                 TxIx        Amount
--------------------------------------------------------------------------------------
56e73127013a279eba46c2fa988caf6982d1f98e04d489b7971925ed23e503b2     0        360000000 lovelace + TxOutDatumHashNone
```

**Review by erikd — DISMISSED** [review]
Can you fix that commit subject line so it does not line wrap.

The subject should just give an overview and the body can contain more details. Probably also worth adding "plutus:" to the start of the subject line just ti make it obvious that this is all that is affected.

**Review by Jimbo4350 — APPROVED** [review]
LGTM!

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**